components/formats-bsd/src/loci/formats/out/TiffWriter.java

```diff
@@ -82,6 +82,9 @@
    */
   private static final long BIG_TIFF_CUTOFF = (long) 1024 * 1024 * 3990;
 
+  /** TIFF tiles must be of a height and width divisible by 16. */
```

Comment:
This was based on the Tiff tag refrence for TileHeight and TileWidth - http://www.awaresystems.be/imaging/tiff/tifftags/tilewidth.html